schemas/Quiz.yaml

```diff
@@ -0,0 +1,17 @@
+title: Quiz
+type: object
+required:
+  - list of questions
```

Comment:
properties should be single words or `camelCase` 
no white spaces in a property name

Reply:
Properties like "title"?